tsconfig.json

```diff
@@ -3,7 +3,7 @@
     /* Basic Options */
     "target": "ES5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
     "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    // "lib": [],                             /* Specify library files to be included in the compilation. */
+    "lib": ["es2019", "dom"],                 /* Specify library files to be included in the compilation. */
```

Comment:
Is `await` for Tone.loaded making us go for the es2019 target? Technically i fine with it, just curious to know.

Reply:
I think we still have ES5 as a compilation target, but it enables ES2019 syntax support & transpilation.

It's not due to async/await (that was OK), this is due to the usage of `Object.fromEntries`. Specifically it's [here](https://github.com/scribbletune/scribbletune/pull/131/files#diff-f9c3b87ffde9e5d4256c51efff988cd5R216).

Reply:
makes sense, that was gonna be my second guess!

Reply:
@qchenevier i discovered an issue and it s probably because of this target. Scribbletune also exports a method for max for live. That functionality now breaks because this target is set to es2019 :/ Basically max for live's JS engine is pretty old and doesnt know what to do with `const`, fat arrows and such. Do we need to to transpile to es2019 specifically? I m open to suggestions on how best to deal with this.